ui-admin/src/util/tableColumnUtils.tsx

```diff
@@ -0,0 +1,17 @@
+import { instantToDefaultString } from '@juniper/ui-core'
+import { ColumnDef } from '@tanstack/react-table'
+
+/**
+ * Creates a column definition for a "created at" column.
+ */
+export function createCreatedAtColumn<T>(): ColumnDef<T> {
```

Comment:
I think the extra create is a bit redundant here (and hard to read).  just like for our mocks we just call them, e.g., `mockSurvey` and not `createMockSurvey()`, I think `createdAtColumn` is fine. 
```suggestion
export function createdAtColumn<T>(): ColumnDef<T> {
```